SIMPLVtkLib/QtWidgets/VSInteractorStyleFilterCamera.cpp

```diff
@@ -1152,6 +1168,109 @@ void VSInteractorStyleFilterCamera::deselectAllFilters()
   }
 }
 
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+void VSInteractorStyleFilterCamera::updateTransformText()
+{
+  QString transformText;
+
+  // Add action type
+  switch(m_ActionType)
+  {
+  case ActionType::Translate:
+	transformText.append("Translating");
+	break;
+  case ActionType::Rotate:
+	transformText.append("Rotating");
+	break;
+  case ActionType::Scale:
+	transformText.append("Scaling");
+	break;
+  case ActionType::ResetTransform:
+  case ActionType::None:
+	break;
+  }
+
+  // Add axis
+  bool axisSelected = false;
+  if (m_ActionType != ActionType::None && m_ActionType != ActionType::ResetTransform)
+  {
+    switch(m_ActionAxis)
```

Comment:
This switch statement does not handle the Axis::None case. If the axis should never ever be set to None at this point, then throw a C++ exception.